public/src/theme.js

```diff
@@ -0,0 +1,50 @@
+import { css } from 'styled-components';
+
+export const theme = {
+  button: {
+    boxShadowSizes: '0px 0px 2px 2px',
+    transitionTiming: '.2s',
+    defaultStyling: css`
+      height: 40px;
+      cursor: pointer;
+      width: 100%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      box-sizing: border-box;
+      text-align: center;
+      font-weight: bold;
+      padding: .25em 1em;
+      position: relative;
+      border-radius: .25em;
+      border-width: 2px;
+      border-style: solid;
+      & + & {
```

Comment:
Generally not a good thing to include in a theme is how two units relate to each other. The theme should define the smallest unit possible. If you wanted a `buttonGroup` component that defined how buttons should relate inside of it, that's more appropriate.

Reply:
Fixed!